docs/core/building/distribution-packaging.md

```diff
@@ -0,0 +1,69 @@
+---
+title: .NET Core distribution packaging
+description: .NET Core distribution packaging
+keywords: .NET, .NET Core, source, build
+author: beleroy
+ms.author: mairaw
+ms.date: 03/29/2017
+ms.topic: article
+ms.prod: .net-core
+ms.devlang: dotnet
+ms.assetid: 71b9d722-c5a8-4271-9ce1-d87e7ae2494d
+---
+
+.NET Core distribution packaging
+================================
+
+As .NET Core becomes available on more and more platforms, thanks to the efforts of dedicated contributors, it is useful to provide guidelines on how to package, name, and version it, in order to ensure a consistency of experience no matter where you choose to run .NET.
+This document provides such guidelines.
+
+What .NET Core is made of
+-------------------------
+
+.NET Core is made of three major parts that need to be packaged:
+
+1. **The host** (also known as the "muxer") has two distinct roles: activate a runtime to launch an application, and activate an SDK to dispatch commands to it. The host is a native executable (`dotnet.exe`) and its supporting policy libraries (installed in `host/fxr`). It's built from the code in the [`dotnet/core-setup`](https://github.com/dotnet/core-setup/) repository. There is typically only one host on a given machine, although that is not a strict requirement.
```

Comment:
is muxer a common word? for me, it just adds confusion here to add that aka but it could just be me

we style paths with italic and use contractions:
that is not -> that isn't